src/test/resources/helper/boolean-or-evaluation.yml

```diff
@@ -0,0 +1,7 @@
+concatenateStrings:
```

Comment:
Sul praegu kaks erinevat stiili steppide nimetamiseks: camelCase ja snake_case. Kuna meil kõik senised confid on olnud snake case'is, siis kasutame pigem seda.